cmd/manager/main.go

```diff
@@ -107,10 +114,32 @@ func main() {
 		os.Exit(1)
 	}
 
-	// Create Service object to expose the metrics port.
-	_, err = metrics.ExposeMetricsPort(ctx, metricsPort)
+	if err = serveCRMetrics(cfg); err != nil {
```

Comment:
Why do we need these in this PR?

Reply:
operator-sdk 0.12.0 has this at generated code, I have updated it to make it compile.